pages/about.vue

```diff
@@ -29,6 +29,7 @@
         <h3>運営者・貢献者</h3>
         <ul>
           <li>運営者: ふぉる (<a href="https://twitter.com/phol_si" target="_blank" rel="noopener">Twitter</a> / <a href="https://github.com/xicri" target="_blank" rel="noopener">GitHub</a>)</li>
+          <li>中国語翻訳データ作成: Bill Haku (<a href="https://twitter.com/Haku_Bill" target="_blank" rel="noopener">Twitter</a> / <a href="https://github.com/Bill-Haku" target="_blank" rel="noopener">GitHub</a>)</li>
```

Comment:
@Bill-Haku I'm adding your name as a contributor to the /about/ page. (See [here](https://zh-translate.genshin-dictionary-auq.pages.dev/about/) for preview page. You need to enter your email address to view the preview page)
Can you check if this is OK? Let me know if you want to change the name or add/remove your social media.

Reply:
I hope you can add my Bilibili account(a streaming media platform, considered as Youtube in China) into it: https://space.bilibili.com/158463764. BTW, my browser says that it cannot open the page from your link. Where should I enter  the email address?
<img width="2006" alt="image" src="https://user-images.githubusercontent.com/48173350/175756312-d5496441-720f-4a80-8f63-87b19bf32582.png">

Reply:
> I hope you can add my Bilibili account(a streaming media platform, considered as Youtube in China) into it: https://space.bilibili.com/158463764.

Thanks, I've added.

> BTW, my browser says that it cannot open the page from your link. Where should I enter  the email address?

Oh, that's strange. I will check the server configuration.

Reply:
I guess your Safari's TLS certificate is not up to date.
Can you check if there are any OS updates available?
Updating your OS and Safari might resolve the problem.

But since it should take a long time, let's skip checking the preview environment this time.